toc/source/conf.py

```diff
@@ -129,7 +129,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+html_title = "UForge Docs latest documentation"
```

Comment:
If you go to http://docs.usharesoft.com/en/latest/
The title is: _Welcome to the UForge Online Documentation - UForge Docs 1.0 documentation_

We can't use the word "latest" in these variables ("version" or "release") in the conf.py because of the format of the variable.
We use the "html_title" var to rewrite the title.

`html_title = "UForge Docs latest documentation"`
Will change the title to: _Welcome to the UForge Online Documentation - UForge Docs latest documentation_

In 3.6 (with 'version' var) it will be: _Welcome to the UForge Online Documentation - UForge Docs 3.6 documentation_

If we use:

`html_title = "UForge Documentation"`
The title will look like: _Welcome to the UForge Online Documentation - UForge Documentation_
Wich is not pretty.